src/lib/swapper/swappers/LifiSwapper/utils/constants.ts

```diff
@@ -6,3 +10,8 @@ export const OPTIMISM_GAS_ORACLE_ADDRESS = '0x4200000000000000000000000000000000
 
 // used for analytics and affiliate fee - do not change this without considering impact
 export const LIFI_INTEGRATOR_ID = 'shapeshift'
+
+export const LIFI_SUPPORTED_CHAIN_IDS: SupportedChainIds = {
+  sell: evmChainIds as unknown as ChainId[],
```

Comment:
Weird having this because of the `as const` and readonly nature but not sure of a better way and this is a safe constant not something programmatic so fine by me!

Reply:
agree, the different `KnownChainIds` and `ChainId` type/enums is pretty annoying in this way too